runsc/fsgofer/filter/config.go

```diff
@@ -26,6 +26,16 @@ import (
 // allowedSyscalls is the set of syscalls executed by the gofer.
 var allowedSyscalls = seccomp.SyscallRules{
 	syscall.SYS_ACCEPT: {},
+	syscall.SYS_SOCKET: []seccomp.Rule{
```

Comment:
Given that most gVisor users do not need UDS connection outside the sandbox, I'd rather not allow socket(2) and connect(2) syscalls by default. How about adding a flag to runsc/main.go (--fsgofer_host_uds) that enabled this feature?
Then localFile.Connect only attempts to connect if the flag is set, and the extra syscalls are added to the set only if the flag is enabled.